Code/autopkglib/SignToolVerifier.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/local/autopkg/python
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""See docstring for SignToolVerifier class"""
+
+import os.path
+import subprocess
+from typing import Any, Dict, List, Optional
+
+from autopkglib import Processor, ProcessorError
+
+__all__ = ["SignToolVerifier"]
+
+
+class SignToolVerifier(Processor):
+    """Verifies an authenticode signed installer using the Microsoft SDK
+    signtool executable."""
+
+    EXTENSIONS: List[str] = [".exe", ".msi"]
+
+    # TODO: How much of this is needed to act as a drop-in replacement in an
+    # override recipe??
+    input_variables: Dict[str, Any] = {
+        "DISABLE_CODE_SIGNATURE_VERIFICATION": {
+            "required": False,
+            "description": ("Prevents this processor from running."),
+        },
+        "input_path": {
+            "required": True,
+            "description": (
+                "File path to an `.msi` or `.exe` file for Authenticode verification",
+            ),
+        },
+        "signtool_path": {
+            "required": False,
+            "description": (
+                "The path to signtool.exe. This varies between versions of the "
+                "Windows SDK, so you can explicitly set that here in an override."
+            ),
+            "default": r"C:\Program Files (x86)\Windows Kits\10\bin\x64\signtool.exe",
```

Comment:
If we're going to eliminate the fallback, this needs to use `os.path.join(os.getenv['PROGRAMFILES']` instead of hardcoding C:.

Reply:
Sounds good to me. I'll update the PR today.

Reply:
Done